docs/source/_static/dark.css

```diff
@@ -4,6 +4,7 @@
     Instructions: Install the extension on a Chromium-based browser
     Then do this to export the CSS: https://git.io/JOM6t and drop it here
     Some color values where manually changed - just search for "/*" in this file and insert them in the new css
+    Some were modified by PTB. Search for PTB.
```

Comment:
Can you unify this? i.e. either change your comments to `/* Manually overridden */` or change those comments to `/* PTB (modified) */`?